### ISSUE TITLE
EPUB3 Media Overlays: how to change highlight color

### ISSUE THREAD

**Issue by vkiranmaniya** [edrlab/thorium-reader#1067]
I want to make a few modification in `-epub-media-overlay-active` CSS class. But I didn't find it. Seems that it's being generated runtime. how do I do that? I've referred to the official [https://www.w3.org/publishing/epub3/epub-mediaoverlays.html](W3C docs MediaOverly) and noticed that class is coming along the audiobook itself. If so, is there any workaround?

**Comment by danielweck** [comment]
The CSS classes associated with the `media:active-class` and `media:playback-active-class` metadata are defined by content creators (not by reading systems), and authored into the package `OPF` XML file:

https://www.w3.org/publishing/epub3/epub-mediaoverlays.html#example-5

https://github.com/IDPF/epub3-samples/blob/0b3e80553f509fbdfe591008cd7d1e804b24db54/30/moby-dick-mo/OPS/package.opf#L36

However, when these authored class definitions are missing in EPUB3 Media Overlays publications, Thorium has a fallback to an arbitrary colour scheme designed to be reasonably legible in sepia and night themes, as well as the typical publisher styles which are usually "black on white".

In summary: Thorium users cannot override authored MO styles, and cannot change the default fallback.

Please let us know if this addresses your question, so we can either close this issue, or leave it open if you would like to further discuss (e.g. feature request, suggestions, etc.)

**Comment by vkiranmaniya** [comment]
@danielweck  Thanks for quick replay.  I also noticed that, got a clue from [https://www.w3.org/publishing/epub3/epub-mediaoverlays.html#sec-docs-assoc-style](Official Docs).
May I know where the fallback specification is?

**Comment by danielweck** [comment]
The EPUB specification does not recommend any particular fallback styling / class names for reading systems implementations.

Note the `MAY` conformance requirement for content creators:

https://www.w3.org/publishing/epub3/epub-mediaoverlays.html#sec-docs-assoc-style
> Visual rendering information for the currently-playing EPUB Content Document element MAY be expressed in the CSS Style Sheet using author-defined classes.

**Comment by danielweck** [comment]
Closing as answered.
